mozilla/fx-private-relay
# MPP-2087 - As a relay user in a non-premium EU country let me know that premium is available in my country so I can get more protections

L10n: https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/151



This PR fixes MPP-2087
 


# New feature description

As a relay user in a non-premium EU country let me know that premium is available in my country so I can get more protections
 
Content and UX: [News UI Templates](https://www.figma.com/file/S8hMIBWphwqDmikvTb3O2T/News-UI-Templates?type=design&node-id=27-220)  

**Acceptance Criteria**

- Show to users in: 
  Cyprus, Estonia, Greece, Latvia, Lithuania, Luxembourg, Malta, Portugal, Slovakia, Slovenia 
 
- Localized for Estonian (et), Greek (el), Latvian (lv), Lithuanian (lt), Portuguese (pt), Slovak (sk), Slovenian (sl) (these are the languages that pontoon supports) 
 
- Show both in add-on and on website.

# Screenshot (if applicable)

![image](https://github.com/mozilla/fx-private-relay/assets/3924990/4b99309a-d3d4-4179-8f6e-bf8ba87ca767)
![image](https://github.com/mozilla/fx-private-relay/assets/3924990/5e873140-d4de-409d-b020-e04e17162646)

# How to test

You need an account without premium. You also need the flag `eu_country_expansion` turned on. Change your language to one of the new countries (Lithuanian, for example). Visit your profile and you should have a new news item. 

# Checklist (Definition of Done)
